CONTRIBUTING.md

```diff
@@ -69,8 +69,7 @@ cycle.
 * Lines that are not part of your change should not be edited (e.g. don't format
   unchanged lines, don't reorder existing imports)
 * Add the appropriate [license headers](#license-headers) to any new files
-* For contributions involving the elasticsearch build you can find (details about the build setup in the
-* [BUILDING](BUILDING.md) file
+* For contributions involving the elasticsearch build you can find (details about the build setup in the [BUILDING](BUILDING.md) file)
```

Comment:
Can you keep the line break, but just remove the `*` from the beginning of line 73?  We limit line lengths in the code base to make things easier to read.